web/src/click_handlers.ts

```diff
@@ -932,6 +932,10 @@ export function initialize(): void {
                 $(e.target).closest("a").length > 0 ||
                 $(e.target).closest(".copy_codeblock").length > 0
             ) {
+                if (document.getSelection()?.type === "Range") {
+                    e.preventDefault();
+                    return;
+                }
```

Comment:
I don't understand this code at all -- it deserves a comment.

What is a sample element when we're in the compose box and selecting a link or copy-codeblock button? Is this about the preview UI? 

I wonder if the outer selectors for this block are stale...